mobile/src/main/java/ch/epfl/sdp/mobile/ui/speech_recognition/ChessDictionary.kt

```diff
@@ -0,0 +1,15 @@
+package ch.epfl.sdp.mobile.ui.speech_recognition
+
+/**
+ * Interface representing the vocabulary of chess move commands
+ * @property chessPieces mapping of chess pieces keywords to their homophones
+ * @property numbers mapping of chess number keywords to their homophones
+ * @property letters mapping of chess letter keywords to their homophones
+ * @property placements all possible combinations of chess board placement
```

Comment:
The `@property` KDoc annotation is used for properties declared in the primary constructor. Please document the properties individually instead.

https://kotlinlang.org/docs/kotlin-doc.html

Reply:
👍